CHANGELOG.md

```diff
@@ -5,11 +5,19 @@ follows a format inspired by [Keep a Changelog](https://keepachangelog.com/en/1.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/panorama-ed/memo_wise/compare/v1.9.0...HEAD)
+## [Unreleased](https://github.com/panorama-ed/memo_wise/compare/v1.10.0...HEAD)
 
 **Gem enhancements:**
 
-- Reduced unzipped gem size from 173KB to 19KB [[#345](https://github.com/panorama-ed/memo_wise/pull/345)]
+_No breaking changes!_
+
+**Project enhancements:**
+
+## [v1.10.0](https://github.com/panorama-ed/memo_wise/compare/v1.9.0...v1.10.0)
+
+**Gem enhancements:**
+
+- Reduced unzipped gem size from 312 kB to 68 kB [[#345](https://github.com/panorama-ed/memo_wise/pull/345)]
```

Comment:
@tagliala I wanted to double-check your numbers but got different results. Here's what I did:

```
$ gem install memo_wise # installs v1.9.0
$ gem info memo_wise
$ du -sh /path/to/ruby/3.3.3/gems/memo_wise-1.9.0
 312K	/Users/jevelyn/.gem/ruby/3.3.3/gems/memo_wise-1.9.0

$ gem build memo_wise.gemspec
$ mv memo_wise-1.10.0.gem ~/Downloads
$ gem unpack ~/Downloads/memo_wise-1.10.0.gem
$ du -sh ~/Downloads/memo_wise-1.10.0
 68K	./memo_wise-1.10.0/
```

did I do something incorrectly?

Reply:
@JacobEvelyn sorry, I just realized that "unzipped" is incorrect

I was referring to the compressed version, I've noticed it was "too big for the source code" here:

https://rubygems.org/gems/memo_wise

Screen attached:

<img width="346" alt="image" src="https://github.com/user-attachments/assets/be3b9c89-6cc6-4d4d-bbe1-a2c520693875">

This will change to 19 KB after the new release

Reply:
Thanks! I see it as 20 kB on my machine for some reason (probably slight differences around the CHANGELOG updates for v1.10.0); I'll change.

Reply:
Thanks, updated screen from rubygems

![Uploading image.png…]()